src/components/AlertBanner/AlertBanner.module.scss

```diff
@@ -25,6 +25,15 @@
   background-color: var(--inverted-color-background);
 }
 
+.warning {
+  background-color: var(--warning-color);
+  color: var(--text-color-fixed-dark);
+
+  .closeBtn {
+    color: var(--text-color-fixed-dark);
+  }
```

Comment:
I preferred to override this since the Button (and IconButton) are too complex to change for this, and if we go towards controlling colors with CSS rather than JS I think it's a good direction anyway

Reply:
agree